spec/httpserver/handlers/ok_spec.clj

```diff
@@ -0,0 +1,11 @@
+(ns httpserver.ok-spec
+  (:require  [speclj.core :refer :all]
+            [httpserver.handlers.ok :as ok]))
+
+(import '(com.td.HttpServer HttpResponse))
+
+(describe "teapotHandler"
+  (it "returns an HttpRequest with code 200"
+    (let  [testOk  (ok/new-ok-handler)
```

Comment:
extra space, and should be kebab cased, and also `test-ok` is kind of a weird variable name, why not `ok-handler`?